src/FakeItEasy/ExceptionMessages.cs

```diff
@@ -118,5 +118,8 @@ public static string CallToUnconfiguredMethodOfStrictFake(IFakeObjectCall call)
 
         public static string NotAnInterface(Type interfaceType) =>
             $"The specified type {interfaceType} is not an interface";
+
+        public static string FailedToFakeNonReferenceType(Type fakeType) =>
+            $"Failed to create Fake of type {fakeType} because it is neither an interface nor a class.";

```

Comment:
nor a delegate
Or maybe just "because it's a value type"?

Reply:
Delegates are classes, no?
Regardless, I'm happier with "because it's a value type".